app/controllers/dashboard/locales_controller.rb

```diff
@@ -0,0 +1,21 @@
+class Dashboard::LocalesController < Dashboard::BaseController
+  before_action :authorize_owner!
+
+  private
+
+  def association_chain
+    current_account.users
```

Comment:
Don’t need to do this. Just override find_resource and return current_user. The you don’t need to call authorize_owner